tensorboard/webapp/notification_center/_data_source/notification_center_data_source.ts

```diff
@@ -32,4 +33,15 @@ export class TBNotificationCenterDataSource
   fetchNotifications(): Observable<NotificationCenterResponse> {
     return this.http.get<NotificationCenterResponse>(`data/notifications`);
   }
+
+  updateAndGetLastReadTimeStampInMs(): Observable<number> {
+    const lasReadTime =
+      window.localStorage.getItem(NOTIFICATION_LAST_READ_TIME_KEY) ?? '-1';
+    const timeNow = Date.now();
+    window.localStorage.setItem(
```

Comment:
FYI, you can have a listener on localStorage so you know when other tabs change the value. You _may_ want this in case you want to dismiss the read dot when user interacts with the bell icon in one of the TensorBoard browser tabs.